virttest/qemu_vm.py

```diff
@@ -2080,11 +2084,9 @@ def add_disable_modern(devices, dev, dev_type):
         for balloon_device in params.objects("balloon"):
             params_balloon = params.object_params(balloon_device)
             balloon_devid = params_balloon.get("balloon_dev_devid")
-            balloon_bus = None
             use_ofmt = params_balloon.get("balloon_use_old_format",
                                           "no") == "yes"
-            if params_balloon.get("balloon_dev_add_bus") == "yes":
-                balloon_bus = pci_bus
+            balloon_bus = {"aobject": params_balloon.get("balloon_bus", "pci.0")}
```

Comment:
Well you can do that by using `pci_bus_image1` or `pci_bus_nic1`. But I see that bounding those might make sense as you might want to put all disks to one pci_bus, the nic interfaces to other and so on. Still the idea of `nic_bus` and `ballon_bus` looks prone to clashes. How about `ballon_pci_bus`, `nic_pci_bus` and so on? I think that would be clearer.